src/schema/schema.graphql

```diff
@@ -3253,6 +3258,7 @@ type Mutation {
 		name: String!
 		parentFolderId: ID
 		url: String
+		isLocalFolder: Boolean
```

Comment:
isLocal?

Reply:
As discussed, Folder and mailItems has different key names to identify whether it's local or not. Also, it is extensively used in zm-x-web. Keeping it as it is.